pkg/airgap/airgap.go

```diff
@@ -265,6 +266,21 @@ func CreateAppFromAirgap(opts CreateAirgapAppOpts) (finalError error) {
 		return errors.Wrap(err, "failed to load kotskinds from path")
 	}
 
+	hasStrictPreflights, err := store.GetStore().HasStrictPreflights(a.ID, newSequence)
+	if err != nil {
+		return errors.Wrap(err, "failed to check if app preflight has strict analyzers")
+	}
+
+	if hasStrictPreflights && opts.SkipPreflights {
+		logger.Warnf("preflights will not be skipped, strict preflights are set to %t", hasStrictPreflights)
+	}
+
+	if !opts.SkipPreflights || hasStrictPreflights {
+		if err := preflight.Run(opts.PendingApp.ID, opts.PendingApp.Slug, newSequence, true, tmpRoot); err != nil {
```

Comment:
hmmm, since this is moved here, let's remove lines 302-307 that sets the version as "pending preflight"? this step is already done in the preflight "Run" function, and we don't wanna end up with a race condition where it's overwritten afterwards.